apps/dotcom/client/e2e/fixtures/Database.ts

```diff
@@ -0,0 +1,11 @@
+import { execSync } from 'child_process'
+import path from 'path'
+
+export class Database {
+	constructor() {}
+
+	async reset() {
+		const scriptPath = path.resolve(__dirname, '../../../sync-worker/')
+		execSync(`${scriptPath}/reset-db.sh > /dev/null 2>&1`, { cwd: scriptPath })
```

Comment:
this doesn't work on it's own, you'd need to restart wrangler too otherwise the durable object will still have all the data in memory.

Reply:
Yeah, good catch. Added a test making sure we really clear it.